atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -332,6 +332,7 @@ private CassandraKeyValueServiceImpl(Logger log,
             CassandraMutationTimestampProvider mutationTimestampProvider) {
         super(AbstractKeyValueService.createFixedThreadPool("Atlas Cassandra KVS",
                 config.poolSize() * config.servers().size()));
+//        mutationTimestampProvider = CassandraMutationTimestampProviders.legacyModeForTestsOnly();
```

Comment:
oops

Reply:
oops!

Reply:
and that's how I tested that things wouldn't work in the old world 😅